cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/LauncherUtils.java

```diff
@@ -167,7 +165,7 @@ public static void runServices(ProgramFile programFile) {
         programFile.setServerConnectorRegistry(serverConnectorRegistry);
         serverConnectorRegistry.initServerConnectors();
 
-        outStream.println("ballerina: initiating service(s) in '" + programFile.getProgramFilePath() + "'");
+        outStream.println("Initiating service(s) in '" + programFile.getProgramFilePath() + "'");
```

Comment:
So we are not going with adding the package name at the front? i.e.., `[ballerina/http]`

Reply:
@pubudu91, this is not a standard library message. This is printed whenever the file/package only has services. We will have to revisit this with the standard library changes.